packages/modelviewer.dev/assets/a11y.json

```diff
@@ -0,0 +1,14 @@
+{
+  "front": "The front of the 3D model",
```

Comment:
Is the idea that you have a separate JSON file for each locality?

Reply:
Found it easier to test just adding a json file that was loaded in. Will implement so that it sends in the json structure directly. I agree that it would be preferable to do that. Not sure if we should keep that you can load in a json structure?

Reply:
Hmm, that wasn't quite what I was referring to. I meant, this change is supposedly to allow multi-language support, right? But the example is all in English. I would like at least two different languages with a button to switch, so that all the code is there for a complete usage of the feature. Probably switch the alt text at the same time. Unless I'm missing something about how localization is generally accomplished on websites?

Reply:
Will add so that there is a examples of multiple languages, that you can switch between on the example page.